### PR TITLE
changed footer message directing to help center article

### LINKED ISSUE #3676: Help-Desk Published: How To Get Blockchain Data From Previous Chain Upgrades
help-center article about accessing data/transactions from previous blockchain upgrades 

https://intercom.help/lunie/en/articles/3787014-how-to-get-blockchain-data-from-previous-chain-upgrades

### PR DESCRIPTION
**Pull request by jrmoreau**
Closes #3676

**Description:**

Changed copy in activity page footer to direct users to help center article on previous chain data retrieval 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/wallet/PageTransactions.vue [hunk 1]
```diff
@@ -25,10 +25,13 @@
...
         </div>
         <div slot="subtitle">
-          Unfortunately, we can't display transactions from previous chains
-          right now.
-          <a class="intercom-button" @click="handleIntercom()">Let us know</a>
-          if you'd like access these transactions.
+          <p>
+            Lunie cannot display transactions from previous chains in your activity page. 
+          </p>
+          <p>if you would like to view information from previous chain upgrades
```
Comment (jbibla) [inline src/components/wallet/PageTransactions.vue]: I try to follow this style guide for links: https://polaris.shopify.com/content/actionable-language#section-links

**Comment by jbibla** [comment]
to get some more green checks you'll have to
- update the changelog with `yarn changelog`
- see why unit tests are failing with `yarn test:unit` (and likely `yarn test:unit -u` to fix snapshots)

**Comment by Bitcoinera** [comment]
James, if you run `yarn test:unit -u` you will get the tests (second failing check) fixed 😉

**Comment by jrmoreau** [comment]
> James, if you run `yarn test:unit -u` you will get the tests (second failing check) fixed 😉

I did this before committing - should I try again?

**Comment by faboweb** [comment]
And as a best practice always provide screenshots of your changes when you alter the UI

**Comment by jrmoreau** [comment]
> And as a best practice always provide screenshots of your changes when you alter the UI

yes sir!

**Pull request merged**